nix/packages/nixos-facter/default.nix

```diff
@@ -62,6 +63,11 @@ in
       "-X github.com/numtide/nixos-facter/pkg/build.System=${pkgs.stdenv.hostPlatform.system}"
     ];
 
+    preCheck = ''
+      HOME=$(mktemp -d)
+      golangci-lint run
```

Comment:
I think we want to move this to another build. If someone pulls the flake with its own nixpkgs override it might fail the build when `golangci-lint run` adds new lints. I usually do overrides for this: https://github.com/Mic92/niks3/blob/ba971512979066830868f71010281a44a2654946/nix/checks/flake-module.nix#L20

Reply:
You make a fair point, I'll make the adjustment